docs/chaosiq-on-docker/run-with-chaosiq.md

```diff
@@ -0,0 +1,67 @@
+
+## Running with ChaosIQ
+
+To connect the image to ChaosIQ when it runs the experiment we will have to provide a settings file. To do this you will need a local environment setup to run the [Chaos Toolkit][ChaosToolkit] with the [ChaosIQ extension][ChaosIQCloud]. For full details of setting this up see [ChaosIQ prerequisites][InstallCtk]. You will also need a Token from ChaosIQ, if you don't already have a token that you can use login to [ChaosIQ][ChaosIQ] and visit the [Tokens page][ChaosIQTokens] to generate a new Token (remember to capture the Token in your clipboard).
+
+To create a settings file for ChaosIQ run the following command:
+
+```bash
+chaos --settings `pwd`/ctk_config/settings.yaml signin
+```
+
+This command will prompt with:
+
+* ChaosIQ url [https://console.chaosiq.io]: *- you can probably accept the default*.
+* ChaosIQ token: *- paste in the Token from ChaosIQ*.
+* Here are your known organizations: *- enter the number for the organization you want to publish to*.
+* Here are the teams you belong to in organization *- enter the number for the team you want to publish to*.
+
+!!!Note
+    If your organization only has one team then the ```signin``` command will automatically select that team.
+
+The ```signin``` command will confirm the settings with something like:
+
+```bash
+Experiments and executions will be published to team 'chaos-ebn' in organization 'ChaosIQ'
```

Comment:
can we have names that make more sense to a larger audience?

Reply:
fixed